components/Brands/brandData.tsx

```diff
@@ -1,48 +1,9 @@
 import { Brand } from "@/types/brand";
 
 const brandData: Brand[] = [
-  {
-    id: 0.25,
-    name: "Client",
-    href: "#",
-    image: "/images/brand/brand-light-01.svg",
-    imageLight: "/images/brand/brand-dark-01.svg",
-  },
-  {
-    id: 0.3,
-    name: "Client",
-    href: "#",
-    image: "/images/brand/brand-light-02.svg",
-    imageLight: "/images/brand/brand-dark-02.svg",
-  },
-  {
-    id: 0.4,
-    name: "Client",
-    href: "#",
-    image: "/images/brand/brand-light-03.svg",
-    imageLight: "/images/brand/brand-dark-03.svg",
-  },
-  {
-    id: 0.5,
-    name: "Client",
-    href: "#",
-    image: "/images/brand/brand-light-04.svg",
-    imageLight: "/images/brand/brand-dark-04.svg",
-  },
-  {
-    id: 0.6,
-    name: "Client",
-    href: "#",
-    image: "/images/brand/brand-light-05.svg",
-    imageLight: "/images/brand/brand-dark-05.svg",
-  },
-  {
-    id: 0.7,
-    name: "Client",
-    href: "#",
-    image: "/images/brand/brand-light-06.svg",
-    imageLight: "/images/brand/brand-dark-06.svg",
-  },
```

Comment:
This is not a good way to remove any component from a next.js project. 
![image](https://github.com/user-attachments/assets/93030d94-0b4e-4e76-9b82-49e25efeaf1f)

YOu can just simply uncomment the <brand> component from the app/page file.